android/app/src/main/kotlin/app/cogwheel/conduit/ConduitPlatformApis.g.kt

```
@@ -4560,4 +4726,24 @@ class PccFlutterApi(private val binaryMessenger: BinaryMessenger, private val me
       }
     }
   }
+  fun onToolCall(callArg: PlatformPccToolCall, callback: (Result<PlatformPccToolResult>) -> Unit)
+{
+    val separatedMessageChannelSuffix = if (messageChannelSuffix.isNotEmpty()) ".$messageChannelSuffix" else ""
+    val channelName = "dev.flutter.pigeon.conduit.PccFlutterApi.onToolCall$separatedMessageChannelSuffix"
+    val channel = BasicMessageChannel<Any?>(binaryMessenger, channelName, codec)
+    channel.send(listOf(callArg)) {
+      if (it is List<*>) {
+        if (it.size > 1) {
+          callback(Result.failure(FlutterError(it[0] as String, it[1] as String, it[2] as String?)))
```

detekt (1.23.8)

[warning] Use the safe cast ('as? String') instead of 'as String?'.

(detekt.potential-bugs.CastToNullableType)